src/components/CIPipelineN/CIPipeline.tsx

```diff
@@ -212,6 +213,10 @@ export default function CIPipeline({
         }
     }, [location.pathname])
 
+    useEffect(() => {
+        validateStage(activeStageName, formData)
+    }, [formData.name])
```

Comment:
Remove this as we don't need useEffect for validating name. Put it in onChange function itself.